src/main/java/uk/gov/hmcts/reform/et/syaapi/helper/PseNotificationHelper.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.reform.et.syaapi.helper;
+
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.et.common.model.ccd.items.PseResponseTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.PseStatusTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.PseResponseType;
+import uk.gov.hmcts.et.common.model.ccd.types.PseStatusType;
+import uk.gov.hmcts.et.common.model.ccd.types.SendNotificationTypeItem;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+public final class PseNotificationHelper {
+    private PseNotificationHelper() {
+    }
+
+    /**
+     * Finds the notification by ID.
+     * @param notifications  - list of all notifications attached to the case
+     * @param notificationId - id of notification we're trying to find
+     * @return the {@link SendNotificationTypeItem} to be updated
+     */
+    public static SendNotificationTypeItem getSelectedNotification(
+        List<SendNotificationTypeItem> notifications,
+        String notificationId) {
+        return notifications.stream()
+            .filter(a -> a.getId().equals(notificationId))
+            .findAny()
+            .orElse(null);
```

Comment:
To make notifications null safe as you can't use stream on an null/empty. Would also need to add the import for it as well
```suggestion
        return emptyIfNull(notifications).stream()
            .filter(a -> a.getId().equals(notificationId))
            .findAny()
            .orElse(null);
```